generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management.component.html.ejs

```diff
@@ -19,15 +19,22 @@
 <div>
     <h2 id="page-heading" data-cy="<%= entityClass %>Heading">
         <span jhiTranslate="<%= i18nKeyPrefix %>.home.title"><%= entityClassPluralHumanized %></span>
-        <%_ if (!readOnly) { _%>
-
-        <button id="jh-create-entity" data-cy="entityCreateButton" class="btn btn-primary float-right jh-create-entity create-<%= entityUrl %>" [routerLink]="['/<%= entityUrl %>/new']">
-            <fa-icon icon="plus"></fa-icon>
-            <span <% if (searchEngine !== false) { %>class="hidden-sm-down" <% } %> jhiTranslate="<%= i18nKeyPrefix %>.home.createLabel">
-            Create a new <%= entityClassHumanized %>
-            </span>
-        </button>
-        <%_ } _%>
+
+        <div class="d-flex justify-content-end">
+          <button class="btn btn-primary mr-2" (click)="handleSyncList()" [disabled]="isLoading">
```

Comment:
can you check the indentation here plz ? It should be with 4 spaces for Angular

Reply:
Yes you're right. I'm checking !